src/plugins/orangehrmClaimPlugin/Api/ClaimRequestActionAPI.php

```diff
@@ -0,0 +1,199 @@
+<?php
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+
+namespace OrangeHRM\Claim\Api;
+
+use Exception;
+use OpenApi\Annotations as OA;
+use OrangeHRM\Claim\Api\Model\ClaimRequestModel;
+use OrangeHRM\Claim\Api\Traits\ClaimRequestAPIHelperTrait;
+use OrangeHRM\Claim\Traits\Service\ClaimServiceTrait;
+use OrangeHRM\Core\Api\V2\Endpoint;
+use OrangeHRM\Core\Api\V2\EndpointResourceResult;
+use OrangeHRM\Core\Api\V2\EndpointResult;
+use OrangeHRM\Core\Api\V2\Exception\ForbiddenException;
+use OrangeHRM\Core\Api\V2\Exception\InvalidParamException;
+use OrangeHRM\Core\Api\V2\Exception\RecordNotFoundException;
+use OrangeHRM\Core\Api\V2\RequestParams;
+use OrangeHRM\Core\Api\V2\ResourceEndpoint;
+use OrangeHRM\Core\Api\V2\Validator\ParamRule;
+use OrangeHRM\Core\Api\V2\Validator\ParamRuleCollection;
+use OrangeHRM\Core\Api\V2\Validator\Rule;
+use OrangeHRM\Core\Api\V2\Validator\Rules;
+use OrangeHRM\Core\Traits\Auth\AuthUserTrait;
+use OrangeHRM\Core\Traits\ORM\EntityManagerHelperTrait;
+use OrangeHRM\Entity\ClaimRequest;
+use OrangeHRM\Entity\Employee;
+use OrangeHRM\Entity\WorkflowStateMachine;
+use OrangeHRM\ORM\Exception\TransactionException;
+
+class ClaimRequestActionAPI extends Endpoint implements ResourceEndpoint
+{
+    use AuthUserTrait;
+    use ClaimRequestAPIHelperTrait;
+    use EntityManagerHelperTrait;
+    use ClaimServiceTrait;
+
+    public const PARAMETER_REQUEST_ID = 'requestId';
+    public const PARAMETER_ACTION = 'action';
+    public const ACTIONABLE_STATES_MAP = [
+        WorkflowStateMachine::CLAIM_ACTION_SUBMIT => 'SUBMIT',
+        WorkflowStateMachine::CLAIM_ACTION_APPROVE => 'APPROVE',
+        WorkflowStateMachine::CLAIM_ACTION_PAY => 'PAY',
+        WorkflowStateMachine::CLAIM_ACTION_CANCEL => 'CANCEL',
+        WorkflowStateMachine::CLAIM_ACTION_REJECT => 'REJECT'
+    ];
+
+    /**
+     * @OA\Put(
+     *     path="/api/v2/claim/requests/{requestId}/action",
+     *     tags={"Claim/Action"},
```

Comment:
```suggestion
     *     tags={"Claim/Requests"},
```
will put it under `Claim/Requests`